graphqlapi/src/main/java/io/stargate/graphql/schema/types/scalars/CustomScalars.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.schema.types.scalars;
+
+import graphql.schema.GraphQLScalarType;
+
+public class CustomScalars {
+
+  public static final GraphQLScalarType UUID =
+      GraphQLScalarType.newScalar()
+          .name("Uuid")
+          .coercing(StringCoercing.UUID)
+          .description(
+              "Represents a CQL `uuid` as a string.\n"
+                  + "Example: \"fe86cad0-1965-11eb-84e6-ab660d2e8c9e\"")
+          .build();
```

Comment:
The `CustomScalar` enum was completely redundant:
* it was just wrapping a `GraphQLScalarType`, without adding any functionality.
* any time an enum constant was referenced, the caller immediately called `getGraphQLScalar()` to get the scalar.
* the code didn't use any other enum features: no `CustomScalar.valueOf()` anywhere, no switch case on the enum, etc.

Making the constants the types directly makes more sense.

Reply:
Agree, enums didn't buy us anything for scalars.